modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/exportimport/content/processor/DLReferencesExportImportContentProcessor.java

```diff
@@ -473,9 +473,16 @@ private String _replaceExportDLReferences(
 
 				exportedReferenceSB.append("[$dl-reference=");
 				exportedReferenceSB.append(path);
-				exportedReferenceSB.append("$,$include-uuid=");
-				exportedReferenceSB.append(
-					dlReferenceParameters.containsKey("uuid"));
+
+				if (dlReferenceParameters.containsKey("friendlyURL")) {
+					exportedReferenceSB.append("$,$include-friendlyURL");
```

Comment:
Shouldn't we follow the same pattern as include-uuid here? `$include-friendly-url=true`

Reply:
On an initial stage I put it as `$include-friendly-url=true` but after a review I though that there is no `$include-friendly-url=false` and has no sense include it.
But if you think that must be this way, I have no strong feelings about one way or another.

let me know about it